sms/send-an-sms.py

```diff
@@ -0,0 +1,20 @@
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+dotenv_path = join(dirname(__file__), '../.env')
+load_dotenv(dotenv_path)
+
+NEXMO_API_KEY = os.environ.get('NEXMO_API_KEY')
```

Comment:
I would usually use `os.getenv` but for some reason that's just personal preference. Just an FYI - happy for you to leave.